src/smtp_server.c

```diff
@@ -122,58 +138,70 @@ int smtp_server_response(bfd_t *f, int code, const char *message)
 
 int smtp_server_process(struct smtp_server_context *ctx, const char *cmd, const char *arg, bfd_t *stream)
 {
-	int schs, continue_session = 1;
+	int disconnect = 1;
+	int hdlr_idx;
+	int code;
+
 	struct smtp_cmd_hdlr_list *hlink;
```

Comment:
This variable is not used and should be removed. It's probably pointed out by the compiler as well - pay attention to compiler warnings.